Gemfile.lock

```diff
@@ -60,7 +60,7 @@ GEM
       rack-test (>= 0.5.4)
       xpath (~> 2.0)
     childprocess (0.8.0)
-      ffi (~> 1.0, >= 1.0.11)
+      ffi (~> 1.0, >= 1.9.24)
```

Comment:
looks like [build](https://travis-ci.org/hacsoc/the_jolly_advisor/builds/473060194#L612) tried install 1.9.18 instead. not super sure why. maybe try setting to that?
```suggestion
      ffi (~> 1.0, >= 1.9.18)
```